modules/java/pages/_partials/gs-additional-steps-for-linux.adoc

```diff
@@ -29,16 +31,22 @@ NOTE: If the setenv.sh file doesn’t exist, you will need to create a new file.
 Desktop::
 +
 --
-. Copy the _native libraries_ (`<pathToCbl>/support`) to a location accessible to -- and usable by -- your build and runtime environments.
-. Add the following libraries to the `LD_LIBRARY_PATH` in the .bashrc file:
-
+. Within the `.zip` file, locate the support files appropriate for your environment.
+. Copy the directory `<pathToCbl>/support` to a location accessible and executable by your build and runtime environments.
++
+include::partial$directory-diagrams.adoc[tag=couchbase-lite-zip]
++
+. Add the path of the directories `libc++` and `libicu` to  `LD_LIBRARY_PATH` in the .bashrc file:
```

Comment:
You'll need the full pathname, here.  Adding "libc++" won't do the trick.  You need:
\<your lib dir\>/support/libc++
\<your lib dir\>/support/libicu

Reply:
Fixed